src/com/palmergames/bukkit/towny/command/NationCommand.java

```diff
@@ -1618,7 +1622,15 @@ public void nationSet(Player player, String[] split) throws TownyException, Inva
 					TownyMessaging.sendErrorMsg(player, e.getMessage());
 				}
 
-			} else if (split[0].equalsIgnoreCase("taxes")) {
+			} else if (split[0].equalsIgnoreCase("spawn")){
+				try{

```

Comment:
This will require a permission node check here
```
if (!TownyUniverse.getPermissionSource().testPermission(player, PermissionNodes.TOWNY_COMMAND_NATION_SET_SPAWN.getNode()))
 						throw new TownyException(TownySettings.getLangString("msg_err_command_disable"));```